src/main/java/slimeknights/tconstruct/gadgets/item/SlimeSlingItem.java

```diff
@@ -44,8 +53,8 @@ public void onPlayerStoppedUsing(ItemStack stack, World worldIn, LivingEntity en
     }
 
     PlayerEntity player = (PlayerEntity) entityLiving;
-    // has to be on ground to do something
-    if (!player.isOnGround()) {
+    // has to be on ground to do something (except Blue)
+    if ((!player.isOnGround() && this.slimeType != StickySlimeBlock.SlimeType.BLUE) || player.isElytraFlying()) {
```

Comment:
Honestly, only green needs to be on the ground. Not even certain green being on ground is a requirement. Consider this check for every type separately.

Notably, I see absolutely no problems with using orange in the air. Purple is a similar usage to blue, so in the air is good there too. Green would be cool to use when you are sliding against a honey block.